modules/rostests/kmtests/ntos_io/IoFilesystem.c

```diff
@@ -140,33 +149,35 @@ TestAllInformation(VOID)
     Length = FIELD_OFFSET(FILE_ALL_INFORMATION, NameInformation.FileName) + MAX_PATH * sizeof(WCHAR);
     Status = QueryFileInfo(FileHandle, (PVOID*)&FileAllInfo, &Length, FileAllInformation);
     ok_eq_hex(Status, STATUS_SUCCESS);
-    if (!skip(NT_SUCCESS(Status) && FileAllInfo != NULL, "No info\n"))
+    if (skip(NT_SUCCESS(Status) && FileAllInfo != NULL, "No info\n"))
```

Comment:
@ThFabba, all your rewrites are fine with me.
Nonetheless, why keep this now-redundant `NT_SUCCESS(Status)` and its related `if (FileAllInfo)`?